### ISSUE TITLE
Update bs5 colors, fonts and basic settings

### ISSUE THREAD

**Issue by vfried** [SwissDataScienceCenter/renku-ui#1261]
We should update the fonts,colors and basic bootstrap settings before we start implementing the redesign.

**Comment by vfried** [comment]
This was partly closed but we need to update the reactstrap from github to a proper release of if with the new bootstrap.

**Comment by ciyer** [comment]
Closed by #1262. If anything new comes up, we will create a new issue.
